omod/src/main/resources/apps/ugandaemr_reports_app.json

```diff
@@ -352,6 +352,14 @@
         "url": "reportingui/runReport.page?reportDefinition=fe02c34a-5c67-453b-93e9-4946e7bd8c59",
         "order": 35
       },
+      {
+        "id": "HMIS 105  4",
```

Comment:
@solemabrothers add Section to be consistent, and also move it to the 2019 monthly reports (See updated reports.json)

Reply:
This is done